src/domain/discounts/details/conditions/edit-condition/edit-condition-modal.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { useContext } from "react"
+import Button from "../../../../../components/fundamentals/button"
+import Modal from "../../../../../components/molecules/modal"
+import LayeredModal, {
+  LayeredModalContext,
+} from "../../../../../components/molecules/modal/layered-modal"
+import { getTitle } from "../../../new/discount-form/edit-conditions-modal"
+import { EditConditionProvider } from "./edit-condition-provider"
+import ProductConditionsTable from "./add-condition-resources/products/product-conditions-table"
+import ProductCollectionsConditionsTable from "./add-condition-resources/collections/collections-conditions-table"
+import ProductTypesConditionsTable from "./add-condition-resources/product-types/type-conditions-table"
+import ProductTagsConditionsTable from "./add-condition-resources/tags/tags-conditions-table"
+import CustomerGroupsConditionsTable from "./add-condition-resources/customer-groups/customer-groups-conditions-table"
+
+type Props = {
+  open: boolean
+  condition?: any
```

Comment:
suggestion: shouldn't we type that and the one bellow with the expected properties at least?

Reply:
Definitely! Must have slipped - thx